alert-common/src/main/java/com/synopsys/integration/alert/common/annotation/SensitiveFieldFinder.java

```diff
@@ -35,16 +32,6 @@ private SensitiveFieldFinder() {
         throw new IllegalStateException("Utility class");
     }
 
-    public static Set<Field> findSensitiveFields(final Class<?> clazz) {
```

Comment:
If this method was the primary reason this class existed, why keep the class around?